src/test/java/com/algorand/indexer/TestIndexerCases.java

```diff
@@ -0,0 +1,45 @@
+package com.algorand.indexer;
```

Comment:
Lets delete this if it's no longer used.

Reply:
removed.